src/components/orders/OrderSurplusDisplay/index.tsx

```diff
@@ -26,12 +36,11 @@ const Surplus = styled.span`
 //   opacity: 0.5;
 // `
 
-export type Props = { order: Order }
+export type Props = { order: Order; amountSmartFormatting?: boolean } & React.HTMLAttributes<HTMLDivElement>
+type SurplusText = { amount: string; percentage: string; amountSmartFormatting: string }
```

Comment:
One last nitpick:

`formatting` makes sense on line 39 since it's a boolean for the act of formatting.
On line 40 though, it shouldn't since it's the formatted amount itself. 
I suggest something like `amountSmartFormatted` or `formattedSmartAmount` or `smartFormattedAmount`. You get the point.

Reply:
Done!